### ISSUE TITLE
Define / implement basic API, possibly mock

### ISSUE THREAD

**Issue by dadaphl** [apeunit/dastack-signaling-scheme-mvp-management#1]
Goal is to develop frontnend and backend in parallel. Please define the Subgraph API for the scheme as early as possible so work on the fronted can begin sooner than later.

**Comment by dadaphl** [comment]
1. Dec. we will either have some mock data (wrong query) or api is implemented

**Comment by plsdlr** [comment]
I got a basic api with a correct subgraph path running will do some more tests this week. Pushed to repo

**Comment by plsdlr** [comment]
The basic and correct api is available. After running: `npm run test -- test/0.0.1-rc.16/UGenericSchemeSignal.spec.ts ` the graphql is available via : 
```
{
      signals{
        id
        data
      }
    }
```

Currently the id refers to the associated DAO Address, which is think makes sense but is obviously up for discussion. Therefore its also possible to query just with the id. I would probally leave this issue open for further discussion.

**Comment by plsdlr** [comment]
On a additional note: Sometimes while testing (mostly when running the test for the first time) i get a 5000ms timeout error. When running the test again directly after this it works fine. At the moment its a bit hard for me to debug this as i think its more something related to testing envioment. If this issue persists i will open a diffrent thread for it.

**Comment by mrrobot16** [comment]
@plsdlr Yeah there is an issue with there production test implementation that it timeouts when running test in github using travis ci I assume. 

We should bring it up to DAOstack folks.
